.zenodo.json

```diff
@@ -56,6 +56,9 @@
       "name": "Esteban, Oscar",
       "orcid": "0000-0001-8435-6191"
     },
+    {
+      "name": "Kalenkovich, Evgenii"
```

Comment:
@kalenkovich Please let me know if you have an affiliation and ORCID you would like associated with your contributions to PyBIDS!

Reply:
@effigies: thank you for adding me to the list. My ORCID is 0000-0002-4606-4179, let's skip the affiliation.

Reply:
```suggestion
      "name": "Kalenkovich, Evgenii",
      "orcid": "0000-0002-4606-4179"
```